lib/styles/components/group-settings.less

```diff
@@ -26,7 +26,7 @@
                 }
             }
             &::before {
-                content: "Change\00000aCover Photo";
+                content: attr(data-coverphototextlineone) " \00000a " attr(data-coverphototextlinetwo);
```

Comment:
I am not sure that this will work. Quote characters look somehow wrong.

Reply:
Please test this.

Reply:
Tested Working fine 
![image](https://user-images.githubusercontent.com/15967809/50917095-be166500-1462-11e9-882b-868153a75c59.png)